config-connector/solutions/projects/kpt/project-hierarchy/README.md

```diff
@@ -0,0 +1,63 @@
+Project Hierarchy
```

Comment:
I may want to mention it again in this doc that users need to install Config Connector and kpt before they move on as they may ignore the readmes from the parent folders.

Reply:
I think that might be a bit overbearing.

Reply:
Okay. SGTM